### PR TITLE
[BUD-213] Feature - Newbie Home Screen

### PR DESCRIPTION
**Pull request by Tonik1204**
# [#213](https://digitalrig.atlassian.net/browse/BUD-213) Feature / Newbie home screen

## Description

- Add Newbie task list screen;
- Add Rounded Header shape;
- Add Loader to Header ;

## Checklist

- [X] Add unit tests;

# Screenshots

Screenshot 2019-12-12 at 10 34 59
Screenshot 2019-12-12 at 10 35 15
Screenshot 2019-12-12 at 10 35 43

### CODE REVIEW

File: src/graphql/contact-details.graphql.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import gql from 'graphql-tag';
 export const NEWBIE_CONTACT_DETAILS = gql`
   query getContactDetails($newbieId: ID!) {
     newbie(newbieId: $newbieId) {
+      id
```
Comment (lucianojss) [inline src/graphql/contact-details.graphql.ts]: Please fix the mocks to remove missing fields warnings

Screenshot 2019-12-13 at 12 46 47

File: src/components/ContactDetails/ContactDetails.tsx [hunk 2]
```diff
@@ -46,8 +46,12 @@ const ContactDetails: React.FC<ContactDetailsProps> = props => {
 
   return (
     <>
-      <Header type={'back'} onButtonClick={onBackClick} />
-      <PageContainer loading={loading} backGroundShape>
+      <Header
+        type={'back' as MenuTypes}
```
Comment (lucianojss) [inline src/components/ContactDetails/ContactDetails.tsx]: Shouldn't we pass directly the enum here? And everywhere the header is imported
`MenuTypes.BACK`

File: src/components/Login/Login.tsx [hunk 3]
```diff
@@ -16,19 +16,16 @@ import DICTIONARY from './dictionary';
...
     margin: '0 auto',
     marginTop: theme.spacing(3),
   },
   submit: {
-    margin: theme.spacing(3, 0, 2),
+    margin: theme.spacing(3, 0),
   },
   container: {
-    marginTop: theme.spacing(9),
+    paddingTop: theme.spacing(10),
```
Comment (lucianojss) [inline src/components/Login/Login.tsx]: 👍

File: src/components/Login/Login.tsx [hunk 4]
```diff
@@ -72,11 +69,7 @@ const Login = () => {
       className={classes.container}
       component='main'
       maxWidth='md'>
-      <Typography
-        className={classes.title}
-        component='h1'
-        variant='h1'
-        align={'center'}>
+      <Typography component='h1' variant='h1' align='center'>
```
Comment (lucianojss) [inline src/components/Login/Login.tsx]: 💯

File: src/components/ContactDetails/ContactDetails.tsx (hunk 2 above)
Comment (Tonik1204) [reply src/components/ContactDetails/ContactDetails.tsx]: Yeah, in all other places it's like that. Good catch, I missed that 👁

File: src/components/Header/Header.tsx [hunk 5]
```diff
@@ -19,57 +28,130 @@ const useStyles = makeStyles<Theme>(theme => ({
...
+        className={clsx(
+          paperAppBarBackground,
+          isRoundedShape && [roundedShape, backgroundLayer]
+        )}
+      />
     </AppBar>
   );
+
+  return isRoundedShape ? (
+    <Box className={roundedAppBarContainer}>
```
Comment (lucianojss) [inline src/components/Header/Header.tsx]: I found out that header is occupying more space than it needs. Take a look

Screenshot 2019-12-16 at 09 08 18

File: src/components/NewbieSelect/__tests__/__snapshots__/NewbieSelect.test.tsx.snap [hunk 6]
```diff
@@ -2,13 +2,66 @@
...
+        onTouchEnd={[Function]}
+        onTouchMove={[Function]}
+        onTouchStart={[Function]}
+        tabIndex={0}
+        type="button"
+      >
+        <span
+          className="MuiIconButton-label"
+        >
+          <svg
```
Comment (lucianojss) [inline src/components/NewbieSelect/__tests__/__snapshots__/NewbieSelect.test.tsx.snap]: Not being mocked.

File: src/components/NewbieTasksList/__tests__/__snapshots__/NewbieTasksList.test.tsx.snap [hunk 7]
```diff
@@ -0,0 +1,212 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Component - TasksList When there are tasks renders correctly 1`] = `
+Array [
+  <div
+    className="MuiBox-root MuiBox-root-13 makeStyles-roundedAppBarContainer-11"
+  >
+    <header
```
Comment (lucianojss) [inline src/components/NewbieTasksList/__tests__/__snapshots__/NewbieTasksList.test.tsx.snap]: not mocked.

File: src/components/TaskDetails/__tests__/__snapshots__/TaskDetails.test.tsx.snap [hunk 8]
```diff
@@ -2,28 +2,124 @@
 
 exports[`Component - TaskDetails renders correctly 1`] = `
 Array [
-  <Header
-    onButtonClick={[Function]}
-    type="back"
-  />,
+  <header
```
Comment (lucianojss) [inline src/components/TaskDetails/__tests__/__snapshots__/TaskDetails.test.tsx.snap]: not being mocked.

File: src/components/TasksList/__tests__/__snapshots__/TasksList.test.tsx.snap [hunk 9]
```diff
@@ -2,12 +2,53 @@
 
 exports[`Component - TasksList When there are tasks renders correctly 1`] = `
 Array [
-  <Header
-    color="paper"
-    onButtonClick={[Function]}
-    type="back"
+  <AppBar
```
Comment (lucianojss) [inline src/components/TasksList/__tests__/__snapshots__/TasksList.test.tsx.snap]: not mocked

File: src/index.css [hunk 10]
```diff
@@ -1,5 +1,6 @@
 body,
 html,
+main,
```
Comment (lucianojss) [inline src/index.css]: this shouldn't be necessary to be here

**Pull request merged**